doc/nrf/releases_and_maturity/releases/release-notes-2.6.1.rst

```diff
@@ -0,0 +1,156 @@
+     - `Changelog <Modem library changelog for v2.6.1_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.6.1_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.6.1`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_261_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+
+Matter
+------
+
+* Updated:
+
+  * The default Message Reliability Protocol (MRP) retry intervals for Thread devices to two seconds to reduce the number of spurious retransmissions in Thread networks.
+  * Increased the number of available packet buffers in the Matter stack to avoid packet allocation issues.
+
+Wi-Fi
+-----
+
+* Added:
+
+  * Build-time disconnection timeout configuration.
+  * BSSID as a parameter for the Wi-Fi ``connect`` command.
+  * Auto-security mode with a fresh key management type to enhance connection time.
+  * The following fixes and improvements:
+
+    * Wi-Fi stuck in an awake state.
+    * Frequent Wi-Fi disconnections in a congested environment and low RSSI.
+    * https upload and download performance issues.
+    * MQTT disconnection in a congested environment and low RSSI.
+    * Selecting the best RSSI access point.
+    * Rate adaptation enhancements to improve range in the 2.4 GHz band.
+    * Radio test - Error in setting ``reg_domain``.
+
+ To accommodate all fixes and improvements, there is a 1 kB increase in the stack usage of WPA supplicant.
+ Users must increase the stack sizes appropriately in their applications.
+
+Thread
+------
+
+* Fixed ``otPlatCryptoPbkdf2GenerateKey`` API implementation to allow a fallback to legacy Mbed TLS implementation instead of returning ``OT_ERROR_NOT_CAPABLE``.
+
+Applications
+============
+
+This section provides detailed lists of changes by :ref:`application <applications>`.
+
+Matter Bridge
+-------------
+
+* Added:
+
+  * The :kconfig:option:`CONFIG_BRIDGE_BT_MAX_SCANNED_DEVICES` Kconfig option to set the maximum number of scanned Bluetooth LE devices.
+  * The :kconfig:option:`CONFIG_BRIDGE_BT_SCAN_TIMEOUT_MS` Kconfig option to set the scan timeout.
+
+* Fixed an issue where the recovery mechanism for Bluetooth LE connections might not work correctly.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+Wi-Fi samples
+-------------
+
+* :ref:`wifi_shell_sample` sample:
+
+  * Modified ``connect`` command to provide better control over connection parameters.
+  * Added ``Auto-Security-Personal`` mode to the ``connect`` command.
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``23cf38934c0f68861e403b22bc3dd0ce6efbfa39``.
```

Comment:
@cvinayak  to confirm the commit

Reply:
Confirmed.

> $ git merge-base upstream/main HEAD
23cf38934c0f68861e403b22bc3dd0ce6efbfa39